core/spec/models/spree/product_spec.rb

```diff
@@ -485,4 +485,28 @@ def build_option_type_with_values(name, values)
     product = Spree::Product.new
     expect(product.master.is_master).to be true
   end
+
+  context '#discontinue!' do
+    let(:product) { create(:product, sku: 'a-sku') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.